lib/run-script-pkg.js

```diff
@@ -59,10 +45,17 @@ const runScriptPkg = async options => {
     return { code: 0, signal: null }
   }
 
-  if (stdio === 'inherit' && banner !== false) {
-    // we're dumping to the parent's stdout, so print the banner
-    console.log(bruce(pkg._id, event, cmd, args))
+  // consuming library decides if this is displayed or not
+  output.standard('')
+  if (pkg._id) {
+    output.standard(`> ${pkg._id} ${event}`)
+  } else {
+    output.standard(`> ${event}`)
+  }
+  if (args.length) {
+    output.standard(`> ${cmd.trim().replace(/\n/g, '\n> ')} ${args.join(' ')}`)
   }
+  output.standard('')
```

Comment:
I think this should still build a multiline string and call `output.standard()` once.